src/cdk-experimental/scrolling/fixed-size-virtual-scroll.ts

```diff
@@ -77,6 +77,13 @@ export class FixedSizeVirtualScrollStrategy implements VirtualScrollStrategy {
   /** @docs-private Implemented as part of VirtualScrollStrategy. */
   onRenderedOffsetChanged() { /* no-op */ }
 
+  /** Scroll to the offset for the given index. */
+  scrollToIndex(index: number, behavior: ScrollBehavior) {
```

Comment:
Add a return type here since it's part of the public API. I think some of the other methods are missing return types as well.

Reply:
Oh this actually doesn't return anything, must've put that by mistake

Reply:
It should still have `void`, though, since dgeni can't use the inferred types. Also missing `@param` JsDocs